components/omega/doc/design/OmegaV1GoverningEqns.md

```diff
@@ -0,0 +1,765 @@
+
+#### Assumptions
+
+For a primitive equation ocean model, we assume the fluid is hydrostatic. For Omega we are not making the Boussinesq assumption, so all density-dependent terms use the full density. In particular, the density coefficient of the pressure gradient in [](#continuous-momentum) is not a constant, as it is in primitive equation models like POP and MPAS-Ocean.
+
+**Hydrostatic:** Beginning with the vertical momentum equation,
+
+$$
+\frac{D w }{D t}
+  =
+  - \frac{1}{\rho} \frac{\partial p}{\partial z} - g + {\bf D}^u + {\bf F}^u
+$$ (continuous-vert-mom)
+
+assume that advection of vertical momentum $Dw/Dt$, dissipation, and forcing are small, and that the first order balance is between pressure gradient and buoyancy,
+
+$$
+\frac{\partial p}{\partial z}
+  = - \rho g.
+$$ (hydrostatic-balance)
+We then integrate from $z$ to the surface $z^{surf}$ to obtain the hydrostatic pressure equation,
+
+$$
+p(x,y,z) = p^{surf}(x,y) + \int_{z}^{z^{surf}} \rho g dz'.
+$$ (continuous-hydrostatic-pressure)
+
+The constitutive equation is the equation of state,
+
+$$
+\rho = f_{eos}(p,\Theta,S).
+$$ (continuous-eos)
+
+where conservative temperature, $\Theta$, and absolute salinity, $S$, are examples of tracers $\varphi$.
+
+The Boussinesq primitive equations also make an incompressibility assumption, which is identical to an assumption of constant density. Non-Boussinesq models do not make thiat assumption and are not explicitly incompressible. However, the mass conservation equation [](continuous-mass), along with an equation of state for sea water where density only varies slightly, results in a fluid that is nearly incompressible.
+
+A concern when using the full, compressible continuity equation is that this might support acoustic waves with wave speeds on the order of 1500 m/s, requiring an extremely small time step. According to [Griffies and Adcroft (2008)](https://agupubs.onlinelibrary.wiley.com/doi/10.1029/177GM18) and [de Szoeke and Samelson (2002)](https://doi.org/10.1175/1520-0485(2002)032%3C2194:TDBTBA%3E2.0.CO;2), the hydrostatic approximation removes vertical sound waves, leaving only barotropic acoustic modes called Lamb waves.  Fortunately, the Lamb waves can be "subsumed" into the external gravity mode because the scale height of the ocean is much larger (200 km) than its depth (~5 km).  This suggests that Lamb waves should not produce any additional constraints on our barotropic time step (though we should keep an eye on this).  For more details on Lamb waves, see [Dukowicz (2013)](https://doi.org/10.1175/MWR-D-13-00148.1)
+
+#### Final Continuous Equations
+
+The final form of the continuous conservation equations for a non-Boussinesq, hydrostatic ocean are
+
+momentum:
+
+$$
+\frac{\partial {\bf u}}{\partial t}
++ \zeta {\bf u}^\perp + f {\bf u}^\perp
++ w\frac{\partial {\bf u}}{\partial z}
+  = - \frac{1}{\rho} \nabla_z p
```

Comment:
We need the geopotential term here and in subsequent terms.  It should not be lumped in F.
```suggestion
  = - \frac{1}{\rho} \nabla_z p - \nabla_z \Phi
```

Reply:
I used $\rho$ in the first half of the document because that is more common in GFD books. I transition to specific volume in the [pressure gradient section](https://portal.nersc.gov/project/e3sm/mpeterse/omega/design-doc-v1-eqns/_build/html/design/OmegaV1GoverningEqns.html#pressure-gradient), and then use $v$ after that.

Reply:
Yes, I think that's fine.